src/service/resource.js

```diff
@@ -265,7 +265,7 @@ export class Resource {
       this.state_ = ResourceState.NOT_LAID_OUT;
     }
     if (opt_callback) {
```

Comment:
Oh! I got it. you mean call `callback(true)` before calling `Resource#changeSize()`?